src/Orleans.Runtime/Catalog/ActivationData.cs

```diff
@@ -842,10 +842,18 @@ private string GetActivationInfoString()
                 String.Format(" #GrainType={0} Placement={1}", GrainInstanceType.FullName, placement);
         }
 
-        public void Dispose()
+        public async ValueTask DisposeAsync()
         {
-            IDisposable disposable = serviceScope;
-            if (disposable != null) disposable.Dispose();
```

Comment:
I think we should make this both IDisposable and IAsyncDisposable

Reply:
I just did. 
But I wonder why this is not the case on master.